### DISCUSSION TITLE
How to customize profile-item icon in panel dropdown header?

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#8994] [Help]
<img width="226" alt="Screenshot 2023-10-09 at 23 01 38" src="https://github.com/filamentphp/filament/assets/121255432/19c0d603-b4bc-4efb-b691-458469549717">

<img width="758" alt="Screenshot 2023-10-09 at 22 59 58" src="https://github.com/filamentphp/filament/assets/121255432/a0560ce6-77ed-4b6e-974b-d9915b2c6d2e">

**Comment by leandrocfe** [comment]
```php
$panel->userMenuItems([
    'profile' => MenuItem::make()->icon('heroicon-o-user'),
])
```

**Reply by devhoussam1998** [reply]
I find the solution at https://filamentphp.com/docs/3.x/support/icons#panel-builder-icon-aliases `panels::user-menu.profile-item` **- Profile item in the user menu**
